packages/cli/src/config/config.ts

```diff
@@ -63,6 +64,11 @@ async function parseArguments(): Promise<CliArgs> {
       description: `Model`,
       default: process.env.GEMINI_MODEL || DEFAULT_GEMINI_MODEL,
     })
+    .option('force-model', {
```

Comment:
we should also probably support force-model in 
settings.ts so you can force it in your settings.json rather than via a command line argument

Reply:
sorry after some further thinking. Can we make this only be in settings.json to start out with?